Reconfiguration of some content in mouse RNA-seq tutorials

mblue9 opened the pull request:
Running an RNA-seq workshop with Melbourne Bioinformatics next week and this PR has some edits we discussed could be good to have (mainly some reorganisation of content). 
Incorporates some of the things mentioned in our previous workshop feedback [here](https://github.com/galaxyproject/training-material/issues/989#issuecomment-493642457)

ping @jessicachung

shiltemann commented on topics/transcriptomics/tutorials/rna-seq-reads-to-counts/tutorial.md:
```diff
@@ -201,6 +218,15 @@ Sequence quality control is therefore an essential first step in your analysis.
 {: .hands_on}
 
+
+> ### {% icon tip %} Tip: Using collections with a tool
```
we have a snippet for this now that you can add in hands-on boxes where relevant {% include snippets/select_collection.md %}

shiltemann reviewed (commented):
awesome, thanks @mblue9!

Some minor suggestions below.
 
These tutorials seem to be designed to follow each other? Could define recommended follow upu trainings in the metadata, in format like this:

```
follow_up_training:
-
  type: "internal"
  topic_name: introduction
  tutorials:
      - r-advanced
``` 

And if you have the time it could be nice to move some of the citations to a tutorial.bib file next to the tutorial, and cite them like {% cite Batut2018 %} (https://doi2bib is useful here). But this can wait until a later PR as well.

shiltemann commented on topics/transcriptomics/tutorials/rna-seq-reads-to-counts/workflows/qc_report.ga:
```diff
@@ -0,0 +1,442 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "",
```
and to fix the failing travis test, please 
1. fill in the tutorial name in this annotation field
2. add a tags attribute: `"tags": ["transcriptomics"]`

mblue9 commented on topics/transcriptomics/tutorials/rna-seq-reads-to-counts/tutorial.md:
```diff
@@ -90,6 +106,7 @@ MCL1-LB	luminalvirgin	https://ndownloader.figshare.com/files/5053585?private_lin
 MCL1-LA	luminalvirgin	https://ndownloader.figshare.com/files/5053552?private_link=f5d63d8c265a05618137
```
Yes great didn't know they were available in Zenodo, I've stolen the reads to counts tut but I'd made some new files for the counts to genes that I'd put in Zenodo so I've kept the link here for that https://zenodo.org/record/4273218

mblue9 commented on topics/transcriptomics/tutorials/rna-seq-counts-to-genes/tutorial.md:
```diff
@@ -106,20 +106,20 @@ We will use three files for this analysis:
 >
 >     To import the files, there are two options:
 >     - Option 1: From a shared data library if available (ask your instructor)
```
The snippet seems to be how to import as datasets not as a collection? or can I change that to collection somehow with `{{ include.astype | default: "as Datasets" }}` I had added a Tip box below that explains how to import from as collection but if you think it can be a snippet I could add that.

mblue9 commented:
Thanks a lot for the quick review and great suggestions @shiltemann ! I think I addressed all your comments but if I should make more changes let me know.

mblue9 commented:
And thanks for the tip about https://doi2bib.org/ ..very useful! I was using a clunky way to get bibtex format before.

shiltemann commented:
thanks @mblue9!
